package-lock.json

```diff
@@ -2357,8 +2357,7 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true,
-          "optional": true
+          "bundled": true
```

Comment:
Why does this file need modification?

Reply:
Removed the package-lock file changes from this branch